pages/common/print.md

```diff
@@ -0,0 +1,25 @@
+# print
+
+> Z Shell (`zsh`) builtin. Prints arguments, similar to `echo`.
+> See also: `echo`, `printf`, `zsh`.
+> More information: <https://zsh.sourceforge.io/Doc/Release/Shell-Builtin-Commands.html>.
+
+- Print input:
+
+`print "Hello" "World"`
+
+- Print separated by newline(s):
+
+`print -l "Linel" "Line 2" "Line3"`
+
+- Print without trailing newline:
+
+`print -n "Hello"; print "World"`
+
+- Enable backslash escapes:
+
+`print -e "Line 1\nLine2"`
+
+- Print arguments as described by 'printf'. Note: For greater portability across shells use `printf`:
```

Comment:
Not sure I understand the `Note:` here? Are you suggesting that users should use the `printf` command instead, or that `print -f` works as a replacement and provides greater portability?

Assuming the latter, then, what about this?
```suggestion
- Print arguments as described by 'printf' (for greater portability across shells, consider instead the `printf` command):
```

Reply:
This comment was set to resolved despite not being addressed.

Reply:
@gsspdev Can you address this issue?